config/initializers/gds_api.rb

```diff
@@ -16,5 +16,3 @@
 Attachable.asset_api_client = GdsApi::AssetManager.new(Plek.current.find('asset-manager'), {
   bearer_token: ENV['PUBLISHER_ASSET_MANAGER_CLIENT_BEARER_TOKEN'] || 'also_set_at_deploy_time',
 })
```

Comment:
Think it might be worth making it clearer that this file is overwritten on deploy. Missed this detail from the existing comment here. Perhaps a separate, more specific comment right at the top.